addon/components/welcome-page.js

```diff
@@ -8,5 +8,16 @@ export default Ember.Component.extend({
     let [ major, minor ] = Ember.VERSION.split(".");
 
     return `${major}.${minor}.0`;
+  }),
+
+  canAngleBracket: Ember.computed(function() {
+    let [ major, minor ] = Ember.VERSION.split(".");
```

Comment:
Let’s pull in ember-compatibility-helpers instead of the Ember.VERSION parsing.

Reply:
I do not know what the problem is but I cannot import `ember-compatibility-helpers`

```
Could not find module `ember-compatibility-helpers` imported from `ember-welcome-page/components/welcome-page`
                at missingModule (http://localhost:7357/assets/vendor.js:252:11)
                at findModule (http://localhost:7357/assets/vendor.js:263:7)
                at Module.findDeps (http://localhost:7357/assets/vendor.js:173:24)
```

I have tried different dependency versions but it always triggers the same error.

Reply:
You put it in dependencies (not devDeps)?

Reply:
@rwjblue I have reported this issue at https://github.com/pzuraq/ember-compatibility-helpers/issues/32.

Perhaps, we could merge this PR as it is now and change it when the issue becomes fixed?

Reply:
The error is shown at https://github.com/ember-cli/ember-welcome-page/pull/127